src/api/index.tsx

```diff
@@ -122,6 +122,21 @@ export const getVersion = async ({
   });
 };
 
+type GetVersionsListParams = {
+  addonId: number;
+  apiState: ApiState;
+};
+
+export const getVersionsList = async ({
+  apiState,
+  addonId,
+}: GetVersionsListParams) => {
+  return callApi<ExternalVersionsList>({
```

Comment:
I'm beginning to wonder if we really need functions to wrap `callApi`. It's just one line of code and still mockable. This is fine for now, just thinking out loud.

Reply:
it makes sense because we can type the return value.

Reply:
But if the thunk executes `callApi<ExternalVersionsList>(...)` directly then it's typed just the same. If we were using that same call in multiple places then it would make sense to have a function but that's the only reason I can think of. That's still a pretty good reason, though. When adding the next call, it may not be obvious to check for existing calls.